tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbStorageTests.cs

```diff
@@ -240,7 +240,7 @@ public async Task CreateObjectTest()
         {
             if (CheckEmulator())
             {
-                await base._createObjectTest(_storage);
+                await base.CreateObjectTest(_storage);
```

Comment:
It's necessary using `base.` here, or can be removed as you did in [MemoryStorage.cs](https://github.com/southworkscom/botbuilder-dotnet/blob/e8f2bb83a05587abde5c0cd667eaf451b5453a4c/tests/Microsoft.Bot.Builder.Tests/MemoryStorageTests.cs#L29)?

Reply:
Yes, we removed it.